extension/src/popup/views/Account.tsx

```diff
@@ -19,6 +19,8 @@ import CopyColor from "popup/assets/copy-color.png";
 import StellarLogo from "popup/assets/stellar-logo.png";
 import { Footer } from "popup/components/Footer";
 
+import "popup/metrics/authServices";
```

Comment:
Couple points I wanted to highlight here:

In the past we've tied metrics logically with reducers, but that always felt strange to me. Tying them to actions makes the API for authoring the metrics emission a little plainer to me, but it means there isn't really anything to export--which I think is fine, but maybe unusual feeling. Since they don't have any impact on functionality, I think it makes sense for them to be something used invisibly.

This _could_ be imported in `App.tsx`, but I deliberately put it with a feature component instead. With code splitting (not a huge concern in an extension but something I wanted to consider for the pattern generally), `import`s can have dramatic impact on how much code is seen as "common" between modules. I find you have to be very deliberate about constructing your dependency tree to get maximum impact. We're not code splitting redux here so this is theoretical, but since this module pulls in the reducer, importing it in `App` would immediately mean all of our app state logic would have to live in the main bundle. For a medium-sized web app, that multiplies per feature and could be a relatively substantial bundle savings.